Healthy/Classes/Generated/UIImage.Generated.swift

```diff
@@ -9,4 +9,16 @@ extension UIImage {
     static var accentColor: UIImage {
       UIImage(named: "AccentColor")!
     }
+
+    static var appColors: UIImage {
+      UIImage(named: "AppColors")!
+    }
+
+    static var icCheckboxNotSelected: UIImage {
+      UIImage(named: "ic_checkbox_not_selected")!
+    }
+
+    static var icCheckboxSelected: UIImage {
+      UIImage(named: "ic_checkbox_selected")!
```

Comment:
```suggestion
      UIImage(named: "icon_checkbox_not_selected")!
    }

    static var icCheckboxSelected: UIImage {
      UIImage(named: "icon_checkbox_selected")!
```

This is more a Swifty way... But thanks for keeping a convention when picking a name for the assets.